src/index.js

```diff
@@ -1,2 +1,22 @@
-/* Nodule OpenAPI.
+/* Opinonated OpenAPI 2.0 (Swagger) Client.
+ *
+ * Design goals:
+ *
+ *  -  Synchronous initialization; asynchronous invocation.
+ *  -  Simple and obvious implementation by default; extensible by design.
  */
+import Client from './client';
+
+
+/* Create a client for a spec and an optional set of options
+ *
+ * Usage:
+ *
+ *    import OpenAPI from 'modules/openapi';
+ *    const client = OpenAPI(spec);
+ *    client.subject.operations(args);
+ *
+ * `spec`: An OpenAPI JSON specificiation. Assuumed to be valid.
+ * `options`: A dictionary of options.
+ */
+export default (spec, name = null, options = {}) => Client(spec, name, options);
```

Comment:
why not `export default Client`?

Reply:
I don't remember any more. I think there used to be a good name and no longer is.